docs/examples.rst

```diff
@@ -377,6 +377,82 @@ Converters are run *before* validators, so you can use validators to check the f
     ValueError: x must be be at least 0.
 
 
+Slots
+-----
+
+By default, instances of classes have a dictionary for attribute storage.
+This wastes space for objects having very few instance variables.
+The space consumption can become acute
+when creating large numbers of instances.
+
+Normal Python classes can avoid using a separate dictionary
+for each instance of a class by defining ``__slots__``.
+``attrs`` classes should just use ``slots=True``.
+
+
+.. doctest::
+
+   >>> @attr.s(slots=True)
+   ... class Coordinates(object):
+   ...     x = attr.ib()
+   ...     y = attr.ib()
+
+
+.. note::
+
+    ``attrs`` slot classes can inherit from other classes
+    just like non-slot classes,
+    but some of the benefits of slot classes are lost if you do that.
+    If you must inherit from other classes,
+    try to inherit only from other slot classes.
```

Comment:
make this just two lines please

I’ll really need to work out something to not send people to misleading documents. 😞